#
#  Copyright (c) 2025 Composiv.ai, Eteration A.S. and others
#
# All rights reserved. This program and the accompanying materials
# are made available under the terms of the Eclipse Public License v2.0
# and Eclipse Distribution License v1.0 which accompany this distribution.
#
# The Eclipse Public License is available at
#    http://www.eclipse.org/legal/epl-v10.html
# and the Eclipse Distribution License is available at
#   http://www.eclipse.org/org/documents/edl-v10.php.
#
# Contributors:
#    Composiv.ai, Eteration A.S. - initial API and implementation
#

/**:
  ros__parameters:
    stack_topic: "stack"
    twin_topic: "twin"
    thing_messages_topic: "thing_messages"
    ignored_packages: [""]

    anonymous: false
    attributes: '{"brand": "muto", "model": "composer"}'
    prefix: muto
    namespace: org.eclipse.muto.sandbox
    name: composer-01